backend/routes/stripe.js

```diff
@@ -209,13 +219,30 @@ module.exports = function (router) {
 
         await stripe.customers.list({ limit: 1 })
 
+        if (!backendUrl) {
```

Comment:
Maybe add a comment explaining why stripeWebhookHost would not be defined?

Reply:
Will add that. BTW, `stripeWebhookHost` is only for development environment, since `localhost` cannot be used for webhooks. It should be `networkConfig.backend` on prod.   
    
Could use Stripe CLI for webhooks instead of registering webhooks while running locally, but this way is closer to how things run on prod. Would be easier to identify/reproduce bugs on local, if any.